sport/app/football/conf/context.scala

```diff
@@ -46,19 +59,24 @@ class FootballLifecycle(
     Ideally, we should combine the logic of these two jobs so we only call once. This is tricky to achieve with the current code.
     We intend to move to a stream model so we can achieve this.
      */
-    jobs.schedule("MatchDayAgentRefreshJob", "0 0/5 * * * ?") {
+    // if preview "Every hour at 40 minutes" otherwise "Every 5 minutes"
+    // 40 minute was chosen for preview to run these jobs after the 35 CompetitionAgentRefreshJobs are finished
+    jobs.schedule("MatchDayAgentRefreshJob", if (context.isPreview) "0 40 * * * ?" else "0 0/5 * * * ?") {
       competitionsService.refreshMatchDay(defaultClock)
     }
 
-    jobs.schedule("MaybeMatchDayAgentRefreshJob", "0 0/1 * * * ?") {
+    // if preview "Every hour at 40 minutes" otherwise "Every minute"
+    jobs.schedule("MaybeMatchDayAgentRefreshJob", if (context.isPreview) "0 40 * * * ?" else "0 0/1 * * * ?") {
       competitionsService.maybeRefreshLiveMatches(defaultClock)
     }
 
-    jobs.schedule("CompetitionRefreshJob", "0 0/10 * * * ?") {
+    // if preview "Every hour at 40 minutes" otherwise "Every 10 minutes"
+    jobs.schedule("CompetitionRefreshJob", if (context.isPreview) "0 40 * * * ?" else "0 0/10 * * * ?") {
```

Comment:
iiuc in preview these jobs will all now run on the 40th minute past the hour, which means they will run at the same time? Perhaps we should stagger them to reduce the chance of all the requests to PA happening at the same time e.g. 40, 45, 50?

Reply:
Sure 1min should be enough to make the request and parse the results